### ISSUE TITLE
Too many USB variables in tests and configs

### ISSUE THREAD

**Issue by johanes2115** [Dasharo/open-source-firmware-validation#47]
Inside platform configs:
- ${USB_STICK}=
- ${WIN_USB_STICK}=
- ${USB_MODEL}=
Plus @{USB_LIST}= with different models in variables.robot. So many variables with the same use case create unnecessary mess.

**Comment by macpijan** [comment]
Yes, the variables must be reworked completely. At first we attack keywords. And during that process, I expect to achieve simplification on the variables front as well.

**Comment by macpijan** [comment]
@BeataZdunczyk Added USB label, we have some USB improvements in progress.

**Comment by macpijan** [comment]
This MR targets to drop USB stick configs from variables.robot and configs as well. Most of the time when we are running regression, the USB stick might be different. Hard-coding the exact USB model have not worked well in practice for us.

https://github.com/Dasharo/open-source-firmware-validation/pull/659
